Dockerfile

```diff
@@ -1,7 +1,7 @@
 #
 # Builder Image
 #
-FROM gradle:5.4-jdk8 AS builder
+FROM gradle:5.6.4-jdk8 AS builder
```

Comment:
This should match the gradle wrapper for best compatibility
https://github.com/spinnaker/kayenta/blob/master/gradle/wrapper/gradle-wrapper.properties#L3

Reply:
- [ ] Use Gradle 6.3